packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -9,26 +9,29 @@ relatedComponents: ['task-list']
 
 ```jsx live
 <ProgressIndicator
+	activePath="#organisations"
 	items={[
-		{ label: 'Introduction', status: 'done', href: '#' },
-		{ label: 'Submit evidence', status: 'saved', href: '#' },
-		{ label: 'Organisations', status: 'started', href: '#' },
-		{ label: 'Business contacts', status: 'error', href: '#' },
-		{ label: 'Case studies', status: 'todo', href: '#' },
-		{ label: 'Attachments', status: 'started', href: '#', isActive: true },
-		{ label: 'Review and submit', status: 'blocked', href: '#' },
+		{ label: 'Introduction', status: 'done', href: '#introduction' },
+		{ label: 'Submit evidence', status: 'saved', href: '#evidence' },
+		{ label: 'Organisations', status: 'started', href: '#organisations' },
+		{ label: 'Business contacts', status: 'error', href: '#contacts' },
+		{ label: 'Case studies', status: 'todo', href: '#case-studies' },
+		{ label: 'Attachments', status: 'started', href: '#attachments' },
+		{ label: 'Review and submit', status: 'blocked', href: '#review' },
 	]}
 />
 ```
 
-Progress indicators are used to show users how many steps are needed to complete a form and where they are in the process of completing it. This helps users know how much is left to do and how far along they are.
+Progress indicators are used to help users navigate to each step of a multi-step form, show users how many steps are needed to complete and where they are in the process of completing it. This helps users know how much is left to do and how far along they are.
```

Comment:
The second sentence feels unnecessary and repetitive of the first.

Also, I wonder if we should use "flow" instead of "form" since, not all pages are form pages (like Review and table file uploads etc)
```suggestion
Progress indicators are used to help users navigate to each step of a multi-step flow, show users how many steps are needed to complete and where they are in the process of completing it.
```

Reply:
@adrian-yee what are your thoughts on @stowball 's copy suggestions? (there are two suggestions)

Reply:
we refer to the confirm and submit page and all secondary actions as part of a multi-step form - please revert to use 'Form':

Progress indicators are used to help users navigate to each step of a multi-step form, show users how many steps are needed to complete a from and how far they have progressed through the process of completing it.

Reply:
- [ ] Docs wording update